osidb/models/tracker.py

```diff
@@ -387,14 +387,20 @@ def _validate_tracker_duplicate(self, **kwargs):
         validate that there is only one tracker with this update stream associated with each affect
         """
         for affect in self.affects.all():
-            if (
-                affect.trackers.filter(ps_update_stream=self.ps_update_stream).count()
-                > 1
-            ):
+            trackers = affect.trackers.filter(ps_update_stream=self.ps_update_stream)
+            if trackers.count() > 1:
+
                 raise ValidationError(
-                    f"Tracker with the update stream {self.ps_update_stream} "
+                    f"Tracker with the update stream {self.ps_update_stream} ({self.external_system_id}) "
                     "is already associated with the affect "
-                    f"{affect.ps_module}/{affect.ps_component} ({affect.uuid})"
+                    f"{affect.ps_module}/{affect.ps_component} ({affect.uuid}) "
+                    f"by the tracker(s) {', '.join([str(tracker.external_system_id) for tracker in trackers if tracker.uuid != self.uuid])}",
+                    params={
+                        "resolution_steps": "When manually cloning a tracker, please ensure that each affect has a unique update stream, "
+                        "which can be identified in the tracker's title. "
+                        "If the tracker is a duplicate, remove the duplicate tracker. "
```

Comment:
By removing the tracker you mean removing the SecurityTracking label so it is not considered a tracker anymore?

Reply:
Good point. This is generally quite complicated and people need a bit of inside to understand what to do. Ideally the tooling would just ignore the duplicates but before it is smart enough manual intervention is needed. I think that a resolution of making sure that every affect's tracker has a unique update stream and contacting us when not sure what to do might be enough :smile: I can imagine various creative ways how people could remove a tracker. I am not sure they would pick removing `SecurityTracking` keyword/label :sweat_smile:

Reply:
Also will removing the label on its own work? I don't think we have a mechanism which will remove the tracker from the database when the label is removed, right?

Reply:
Good point. I know I implemented one for the flaws in the past but I am not about the trackers